services/init-protonet.service

```diff
@@ -27,7 +27,7 @@ ExecStartPre=/bin/mkdir -p /data/soul/public
 ExecStartPre=/bin/mkdir -p /data/soul/uploads
 ExecStartPre=/bin/mkdir -p /data/soul/protosync
 ExecStartPre=/bin/mkdir -p /data/pgp
-ExecStartPre=/bin/bash -c '[[ ! -d /backup ]] && zfs create -p -o mountpoint=/backup protonet_storage/backup'
+ExecStartPre=-/bin/bash -c '[[ ! -d /backup ]] && zfs create -p -o mountpoint=/backup protonet_storage/backup'
```

Comment:
It seems like `zfs create` does nothing if a filesystem already exists and it exits with 0, so `/usr/sbin/zfs create -p -o mountpoint=/backup protonet_storage/backup` would be sufficient.